lighthouse-core/scripts/legacy-javascript/main.js

```diff
@@ -4,6 +4,8 @@
 class MyTestClass {};
 async function reg(...args) {
   await 1;
+  for (let i = 0; i < 10; i++) await 2;
```

Comment:
is this loop significant?
```suggestion
  for (let i = 0; i < 10; i++) await 2; // loop to force polyfill for BLAH
```

Reply:
It simply makes more interesting output for the regenerator transform. It's not necessary except that when I eyeballed the output I was able to tell how it worked better.

Comment seems unnecessary given that :)